src/Basics.lidr

```diff
@@ -291,10 +298,10 @@ definition, and `infixl` specifies left-associative fixity.
 
 === Exercises: 1 star (nandb)
 
-Fill in the hole `?nandb_rhs` and omplete the following function; then make sure
-that the assertions below can each be verified by Idris. (Fill in each of the
-holes, following the model of the `orb` tests above.) The function should return
-`True` if either or both of its inputs `False`.
+Fill in the hole `?nandb_rhs` and complete the following function; then make 
+sure that the assertions below can each be verified by Idris. (Fill in each of 
+the holes, following the model of the `orb` tests above.) The function should 
+return `True` if either or both of its inputs `False`.
```

Comment:
It seems like we should probably reword "... inputs `False`." Maybe something like:
> if either of its inputs is `False`